test/unit/lists-component.spec.ts

```diff
@@ -563,3 +569,160 @@ describe('when deleting a list', () => {
     })
   })
 })
+
+describe('when getting a list', () => {
+  describe('and the list was not found or was not accessible by the user', () => {
+    let error: Error
+
+    beforeEach(() => {
+      error = new ListNotFoundError(listId)
+      dbQueryMock.mockResolvedValueOnce({ rowCount: 0 })
+    })
+
+    it('should throw a list not found error', () => {
+      return expect(listsComponent.getList(listId, { userAddress })).rejects.toEqual(error)
+    })
+  })
+
+  describe('and the user has no access to the list', () => {
+    let error: Error
+
+    beforeEach(() => {
+      error = new ForbiddenAccessToList(listId)
+      dbQueryMock.mockResolvedValueOnce({ rowCount: 1, rows: [{ id: listId, user_address: 'anotherUserAddress' }] })
+    })
+
+    it('should throw a list not found error', () => {
+      return expect(listsComponent.getList(listId, { userAddress, requiredPermissions: [Permission.VIEW] })).rejects.toEqual(error)
+    })
+  })
+
+  describe('and neither the default list nor the permissions should be considered', () => {
+    let dbList: DBList
+    let result: DBList
+
+    beforeEach(async () => {
+      dbList = {
+        id: 'aListId',
+        name: 'aListName',
+        description: null,
+        user_address: 'aUserAddress',
+        created_at: new Date()
+      }
+
+      dbQueryMock.mockResolvedValueOnce({ rowCount: 1, rows: [dbList] })
+      result = await listsComponent.getList(listId, { userAddress, considerDefaultList: false })
+    })
+
+    it('should have made the query to get without checking if the list belongs to the default user or if has the required permissions', async () => {
+      expect(dbQueryMock).toHaveBeenCalledWith(
+        expect.objectContaining({
+          text: expect.stringContaining('SELECT *')
+        })
+      )
+
+      expect(dbQueryMock).toHaveBeenCalledWith(
+        expect.objectContaining({
+          text: expect.stringContaining('FROM favorites.lists WHERE id = $1 AND (user_address = $2)'),
+          values: expect.arrayContaining([listId, userAddress])
+        })
+      )
+    })
+
+    it('should resolve', () => {
```

Comment:
What if we add here the `with the list` phrase to make the assertion clearer?
```suggestion
    it('should resolve with the list', () => {
```